lib/bundler/cli/cache.rb

```diff
@@ -12,7 +12,7 @@ def run
       setup_cache_all
       Bundler.settings[:cache_all_platforms] = options["all-platforms"] if options.key?("all-platforms")
       Bundler.load.cache
-      Bundler.settings[:no_prune] = true if options["no-prune"]
```

Comment:
@segiddins the easiest way would be `Bundler.settings[:no_prune] = !options["prune"]`

or do I need to consider that this option can be set by something else (like a bundler config file)? If that's the case I need to explicitly check if the option is given to set it. If it is given it sets it to the `!options["prune"]` value. If it's not given it defaults to `true` if the settings is not set already